packages/eslint-config-react-native-community/index.js

```diff
@@ -112,7 +112,6 @@ module.exports = {
 
   rules: {
     // General
-    'comma-dangle': [1, 'always-multiline'], // allow or disallow trailing commas
```

Comment:
It seems like we prefer to leave these, but just change them to 0

Reply:
Cool np

Reply:
@TheSavior Changes made